api/v1alpha1/envoyproxy_types.go

```diff
@@ -67,6 +67,12 @@ type EnvoyProxySpec struct {
 	// +optional
 	Concurrency *int32 `json:"concurrency,omitempty"`
 
+	// ExtraArgs defines additional command line options that are provided to Envoy.
+	// More info: https://www.envoyproxy.io/docs/envoy/latest/operations/cli#command-line-options
+	//
```

Comment:
can you add a warning message about duplicate arg?

Reply:
We can tell users which args can/can't be configured through `ExtraArgs` in the comment here. So they won't have to try and guess.

Reply:
Should I add a generic message or specify all internal args that can't be used(`--service-cluster`, `--service-node`, `--log-level`, `--conponent-log-level`, `--concurrency`, `--config-yaml`, `--cpuset-threads`)?

Reply:
I think it's fine to not list all of them.

Reply:
That's what I thought, thanks

Reply:
Added

Reply:
It may be useful to tell users which args can/can't be configured through `ExtraArgs` in the comment here. So they won't need to try and guess. And we can add validation here as well.